Formula/gerbil-scheme.rb

```diff
@@ -47,6 +47,15 @@ def install
     end
   end
 
+  def caveats; <<~EOS
+    You'll need to add to the Gerbil directory to your PATH
+     $ export PATH=/usr/local/opt/gambit-scheme/current/bin:$PATH
+
+    Also setup GERBIL_HOME to find the proper libraries
+     $ export GERBIL_HOME=/usr/local/opt/gerbil-scheme/libexec
```

Comment:
Ideally we'd use a wrapper script to set these variables for the user so they don't have to.

Reply:
This is taken directly from the Readme in the Gerbil repository.

Reply:
https://github.com/vyzo/gerbil/blob/master/doc/guide/README.md#installation-on-macos for reference.

Reply:
Yes but instead of telling people to set it manually we can do it for them with e.g. `bin.env_script_all_files(libexec/"bin", :GERBIL_HOME => "#{libexec}")`

Reply:
Perfect. will add, thanks

Reply:
We would not need to do the symlinks then with this?

Reply:
@ober Yes, that means that the manual symlink creation can be removed.